scenarios/kubernetes_bazel.py

```diff
@@ -128,7 +128,9 @@ def main(args):
         build_pkgs = None
         test_pkgs = None
         if args.build:
-            build_pkgs = args.build.split(' ')
+            # TODO(krzyzacy): hack for 1.6, remove once we don't support 1.6
```

Comment:
hm, this hack doesn't really fix the issue that we'd also like to use a different set of targets on master (since we can now build everything).

wondering if we should just create a dict of build/test targets for each branch in the scenario, and then reference that from the config, similar to what we do with the kubernetes_kubelet scenario.

Reply:
@krzyzacy how far is kubetest from being able to replace a scenario like this?